docs/upgrade-to-v5.md

```diff
@@ -130,3 +130,7 @@ Now `-0` can be used directly
 BSON.deserialize(BSON.serialize({ d: -0 }))
 // type preservation, returns { d: -0 }
 ```
+
+### Extended JSON `strict` flag removed
+
+Extended JSON `parse` and `stringify` APIs ignore the deprecated `strict` option and it has been removed from the type definitions.
```

Comment:
This is not clear.  Did they used to ignore the option or do they now?  We should make the difference between the old and new behaviors explicit.

Maybe something like:
```
In BSON 4x, the `strict` option is deprecated in the extended JSON `parse` and `stringify` functions.  

In BSON 5x, this option is removed.
```